### ISSUE TITLE
loss bbx is nan when train from scratch

### ISSUE THREAD

**Issue by Hlings** [open-mmlab/mmdetection#5581]
I want to train faster rcnn in VOC dataset which just has one person class. Model configs and datasets configs have been changerd correct. But just loss_bbx will be nan while training from scratch. I can't deal with this problem by chaning learning rate or batchsize. Thanks.

**Comment by Hlings** [comment]
here is the log content:
2021-07-10 20:51:25,495 - mmdet - INFO - Start running, host: root@b6264dc00e17d011eb0b78b029bb3a17e6fc-gaoyp131-0, work_dir: /userhome/mmdetection/work_dirs/voc_30
2021-07-10 20:51:25,495 - mmdet - INFO - workflow: [('train', 1)], max: 48 epochs
2021-07-10 20:53:14,106 - mmdet - INFO - Epoch [1][50/115]	lr: 4.995e-05, eta: 3:17:46, time: 2.169, data_time: 0.230, memory: 29025, loss_rpn_cls: 0.6867, loss_rpn_bbox: 0.0241, loss_cls: 0.6853, acc: 52.8721, loss_bbox: nan, loss: nan
2021-07-10 20:54:42,037 - mmdet - INFO - Epoch [1][100/115]	lr: 9.990e-05, eta: 2:57:24, time: 1.759, data_time: 0.037, memory: 29025, loss_rpn_cls: 0.6819, loss_rpn_bbox: 0.0256, loss_cls: 0.5781, acc: 98.0828, loss_bbox: nan, loss: nan
2021-07-10 20:55:08,787 - mmdet - INFO - Saving checkpoint at 1 epochs
2021-07-10 20:55:49,393 - mmdet - INFO - 
+--------+------+--------+--------+-------+
| class  | gts  | dets   | recall | ap    |
+--------+------+--------+--------+-------+
| person | 4528 | 200700 | 0.292  | 0.002 |
+--------+------+--------+--------+-------+
| mAP    |      |        |        | 0.002 |
+--------+------+--------+--------+-------+
2021-07-10 20:55:49,402 - mmdet - INFO - Exp name: voc_30_cnn.py
2021-07-10 20:55:49,402 - mmdet - INFO - Epoch(val) [1][115]	AP50: 0.0020, mAP: 0.0021
2021-07-10 20:57:28,887 - mmdet - INFO - Epoch [2][50/115]	lr: 1.648e-04, eta: 2:39:55, time: 1.986, data_time: 0.232, memory: 29025, loss_rpn_cls: 0.6623, loss_rpn_bbox: 0.0231, loss_cls: 0.4769, acc: 98.2568, loss_bbox: nan, loss: nan
2021-07-10 20:58:56,301 - mmdet - INFO - Epoch [2][100/115]	lr: 2.148e-04, eta: 2:37:32, time: 1.749, data_time: 0.033, memory: 29025, loss_rpn_cls: 0.6243, loss_rpn_bbox: 0.0275, loss_cls: 0.4856, acc: 98.6785, loss_bbox: nan, loss: nan
2021-07-10 20:59:21,325 - mmdet - INFO - Saving checkpoint at 2 epochs
2021-07-10 20:59:58,847 - mmdet - INFO -

**Comment by AKMourato** [comment]
This can happen when the loss functions degenerate. Try to reduce the learning rate.

**Comment by Hlings** [comment]
> This can happen when the loss functions degenerate. Try to reduce the learning rate considerably.
> Another possibility has to do with the bounding boxes annotations, check if all the instances have positive areas.

Thank you very much! The reason is wrong bounding boxes label during data processing.
